src/Config/LibGSupportedReference.props

```diff
@@ -0,0 +1,16 @@
+<Project  ToolsVersion="15.0"  xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <ItemGroup>
+        <PackageReference Include="DynamoVisualProgramming.LibG_225_0_0" GeneratePathProperty="true">
+            <Version>2.9.*</Version>
```

Comment:
@mjkkirschner so the idea is to set this to a specific version manually for say an RC branch or just before release?

Reply:
> @alfredo-pozo it looks good to me, can I ask though, why not just add these props to the existing cs props file thats already imported?
> 
> Do you think it's more explicit whats happening this way?
> 
> ah - I see I guess this way it's also not adding that import to every project?

That's correct @aparajit-pratap , it will be only added to the places where it's needed

Reply:
@aparajit-pratap I was thinking we'd do it as "*" normally and then change to a specific version just before release - but either approach is fine I think, in either case we just need to remember to change these 3 vars.

Reply:
Let me know if you want me to switch to "*" @mjkkirschner

Reply:
Switched to "*" @mjkkirschner